flutter/todo_cubit_wasm/sh/wasm-release

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT=$DIR/..
+
+cargo build --release --target wasm32-unknown-unknown --lib --target-dir target_wasm 
+
+# https://rustwasm.github.io/book/reference/code-size.html#use-the-wasm-opt-tool
```

Comment:
This one should be changed to `https://rustwasm.github.io/docs/book/reference/code-size.html#use-the-wasm-opt-tool`, same as above.